src/help/util.ts

```diff
@@ -38,24 +38,20 @@ export function template(context: any): (t: string) => string {
 function collateSpacedCmdIDFromArgs(argv: string[], config: IConfig): string[] {
   if (argv.length === 1) return argv
 
-  const ids = new Set(config.commandIDs.concat(config.topics.map(t => t.name)))
-
+  const ids = config.collectUsableIds()
   const findId = (argv: string[]): string | undefined => {
     const final: string[] = []
-    const idPresent = (id: string) => ids.has(id)
+    const idPresent = (id: string) => ids.includes(id)
     const isFlag = (s: string) => s.startsWith('-')
     const isArgWithValue = (s: string) => s.includes('=')
     const finalizeId = (s?: string) => s ? [...final, s].join(':') : final.join(':')
 
     const hasSubCommandsWithArgs = () => {
       const id = finalizeId()
-      /**
-       * Get a list of sub commands for the current command id. A command is returned as a subcommand under either
-       * of these conditions:
-       * 1. the `id` start with the current command id.
-       * 2. any of the aliases start with the current command id.
-       */
-      const subCommands = config.commands.filter(c => (c.id).startsWith(id) || c.aliases.some(a => a.startsWith(id)))
+      if (!id) return false
+      // Get a list of sub commands for the current command id. A command is returned as a subcommand if the `id` starts with the current command id.
```

Comment:
Does part 2. of the deleted comment still apply? If so can we keep it please?

Reply:
It no longer applies since aliases are now automatically loaded up in the list of commands now